docs/api/synthetics/private-locations/create-private-location.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[create-private-location-api]]
+== Create Private Location API
+++++
+<titleabbrev>Create Private Location</titleabbrev>
+++++
+
+Creates a private location with the following schema.
+
+=== {api-request-title}
+
+`POST <kibana host>:<port>/api/synthetics/private-locations`
+
+`POST <kibana host>:<port>/s/<space_id>/api/synthetics/private-locations`
+
+=== {api-prereq-title}
+
+You must have `all` privileges for the *Synthetics* feature in the *{observability}* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+[[private-location-request-body]]
+==== Request body
+
+The request body should contain the following attributes:
+
+`label`::
+(Required, string) A label for the private location.
+
+`agentPolicyId`::
+(Required, string) The ID of the agent policy associated with the private location.
+
+`tags`::
+(Optional, array of strings) An array of tags to categorize the private location.
+
+`geo`::
+(Optional, object) Geographic coordinates for the location. It should include the following attributes:
+
+- `lat` (Required, number): The latitude of the location.
+- `lon` (Required, number): The longitude of the location.
+
+[[private-location-create-example]]
+==== Example
+
+Here is an example of a POST request to create a private location:
+
+[source,sh]
+--------------------------------------------------
+POST /api/private-locations
+{
+  "label": "Private Location 1",
+  "agentPolicyId": "abcd1234",
+  "tags": ["private", "testing"],
+  "geo": {
+    "lat": 40.7128,
+    "lon": -74.0060
+  }
+}
+--------------------------------------------------
+
+The API returns the created private location as follows:
```

Comment:
This description is correct and the `POST` endpoint should only return the created resource. However, currently the implementation returns an array of all the Private Locations. The endpoint should be updated, also UI should be checked if it depends on the returned array against the `POST` endpoint.